BlocSpot/BlocSpot/ListViewController.m

```diff
@@ -21,14 +23,17 @@ @implementation ListViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+}
 
+-(void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray *)locations {
+    
+    CLLocationCoordinate2D initialLocation = [locations[0] coordinate];
+    MKCoordinateRegion initialRegion = MKCoordinateRegionMakeWithDistance(initialLocation, 8, 8);
+    
     self.data = [[MapItemData alloc] init];
-    [self.data returnMapItems:^(NSArray *mapItems, NSString *text) {
-        for (MKMapItem *item in mapItems) {
-            [self.returnMapItems addObject:item];
-        }
-        [self.listTableView reloadData];
-    }];
+    [self.data returnMapItems:^(NSArray *mapItems) {
```

Comment:
Should there be some code running during this block? The only action I would want to take during this block is to populate the text for the tableView, but wouldn't that occur in the tableView method?